javatests/arcs/core/storage/StorageProxyTest.kt

```diff
@@ -873,6 +874,7 @@ class StorageProxyTest {
         withTimeout(timeout) { receive() }
 
     private fun runTest(block: suspend CoroutineScope.() -> Unit) = runBlocking {
+        ArcsStrictMode.disableStrictHandles()
```

Comment:
We don't want them enabled here?

Reply:
Because the tests are written to test StorageProxy operations outside of a Scheduler. so either 
1) disable or
2) the tests are wrong, and StorageProxy test methods should be updated to run within a dispatcher? WDYT?

Reply:
It's probably fine to leave as-is for now, could you add a TODO with a bug to come back and enable strict mode here on an as-needed basis (test-by-test)?

Reply:
Will Do, thanks for the quick review, enjoy your vacation and stay safe!